corehq/apps/users/static/users/js/filtered_download.js

```diff
@@ -18,26 +18,31 @@ hqDefine('users/js/filtered_download', [
         };
     }
 
+    function countUsers(currentFilters) {
+        var countUsersUrl = initialPageData.get('count_users_url');
+        $.get({
+            url: countUsersUrl,
+            data: currentFilters,
+            success: function (data) {
+                var count = data.count;
+                var template = count === 1 ? gettext("Download <%= count %> user") : gettext("Download <%= count %> users");
+                $('.submit_button').text(_.template(template)({count: count}));
+            },
+            error: function () {
+                alert("Error determining number of matching users");
+            },
+        });
+    }
+
     $(function () {
         var prevFilters = getFilters();
-        var countUsersUrl = initialPageData.get('count_users_url');
         setInterval(function () {
```

Comment:
Realize you didn't introduce this, but...what would you think of making this trigger when there's a change event, instead of polling every second? We've seen problems recently with pages that poll frequently interfering with login when they're left open for a while.

Reply:
Well I don't think this page is actually really used by non-admins so it should not cause issue as other polls but nevertheless worth changing.
https://github.com/dimagi/commcare-hq/pull/27625/commits/4b54d43e1589cc89656125f89463418cd7810456

Reply:
Admins also get unhappy when they can't log in ;)

Reply:
Oh, I thought the polling would generally be causing load on resources and may be slowness of the page. It went to the extent of hindering login! 
I was just saying that this page won't contribute much to that load because it's hardly used but we got this covered now.